src/components/KonnectorSuccess.spec.jsx

```diff
@@ -1,18 +1,39 @@
 import React from 'react'
-import { mount } from 'enzyme'
-import KonnectorSuccess, { BanksLink, DriveLink } from './KonnectorSuccess'
+import { render, screen } from '@testing-library/react'
+
 import AppLike from 'test/AppLike'
+import KonnectorSuccess from './KonnectorSuccess'
+
+// @TODO: some pretty ugly mocks here because the app uses different react-redux versions
```

Comment:
Can you tell which versions do we have? Can't we upgrade to react-redux 7.2.7 (Cozyclient's one)?

Can't we add a deduplicate plugin to only get 1 version?

Reply:
I tried to upgrade to react-redux@8 (latest), but it broke many things. I can try again.

The issue is between cozy-client and cozy-home, their versions are different, and not only different but incompatible because they aren't using the same API. I don't think we can remove the duplication here

Reply:
I did the check, this is what messed up my env. I updated react-redux when trying to make the tests works. But then the app failed at runtime too. I didn't realize that those were two different errors.